mappings/net/minecraft/entity/ai/brain/task/ForgetAttackTargetTask.mapping

```diff
@@ -2,3 +2,11 @@ CLASS net/minecraft/unmapped/C_gaizicad net/minecraft/entity/ai/brain/task/Forge
 	FIELD f_npnuwhrk REMEMBER_TIME I
 	METHOD m_agdthaky cannotReachTarget (Lnet/minecraft/unmapped/C_usxaxydn;Ljava/util/Optional;)Z
 		ARG 0 entity
+	METHOD m_nwhekhlv run (Ljava/util/function/Predicate;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 predicate
+	METHOD m_nyujuwga run ()Lnet/minecraft/unmapped/C_mdnathub;
+	METHOD m_pdsgbbnj run (Ljava/util/function/Predicate;Ljava/util/function/BiConsumer;Z)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 predicate
+		ARG 1 biConsumer
```

Comment:
it isn't immediately clear what this is consuming, so we can't just name it "consumer"